### PR TITLE
#Provide intro for create-react-native-app and react-native init

### PR DESCRIPTION
**Pull request by probeadd**
Reference pull request: #2206 
Here I try to provide intro using both create-react-native-app and react-native init.

### CODE REVIEW

**Comment by probeadd** [comment]
@matthamil @davidtrogers How about this? I have provide both for CRNA and react-native init. Thank you.

**Comment by nonameolsson** [comment]
Good!

File: docs/guides/Guide-Intro.md [hunk 1]
```diff
@@ -20,6 +20,21 @@ react-native run-android # or:
...
+create-react-native-app SimpleApp
+cd SimpleApp
+
+# Install the latest version of react-navigation from npm
+npm install --save react-navigation
+
+# Run the new app
+npm start
+
+#This will start a development server for you, and print a QR code in your terminal.
```
Comment (matthamil) [inline docs/guides/Guide-Intro.md]: nit: remove the comma and add a space between `#` and `This`.

File: docs/guides/Guide-Intro.md [hunk 2]
```diff
@@ -55,6 +70,7 @@ const SimpleApp = StackNavigator({
   Home: { screen: HomeScreen },
 });
 
+//if you are using create-react-native-app you don't need this line
```
Comment (matthamil) [inline docs/guides/Guide-Intro.md]: Nit: space between `//` and `if`

**Comment by spencercarli** [comment]
Merged #2491

**Pull request closed without merging**